src/postprocess/band_structure.jl

```diff
@@ -135,14 +135,24 @@ function plot_band_data(band_data; εF=nothing,
     p
 end
 
+function detexify_kpoint(string)
+    # For some reason Julia doesn't support this naively: https://github.com/JuliaLang/julia/issues/29849
+    replacements = ("\\Gamma" => "Γ",
+                    "\\Delta" => "Δ",
+                    "\\Sigma" => "Σ")
+    for r in replacements
+        string = replace(string, r)
+    end
+    string
+end
```

Comment:
Is there really no shorter way to do this ... that's a lot of code for just 3 replacements.

Reply:
I agree! But I don't see a faster way (except for some fold trickery). We can just remove this...